### DISCUSSION TITLE
Make user on server without registering through email

### DISCUSSION THREAD

**Discussion by onyxiarivera** [asciinema/discussions#108] [Q&A]
Is it possible to make a user file somewhere or some sort of bypass that allows signup without an email address? I don't have any actual domains attached to the VPS running the Docker server image, and I don't plan on doing so unless absolutely necessary.

**Comment by ku1ik** [comment]
I see your point. Creating the user won't solve authentication problem alone, as you'd still need to receive email for subsequent logins.

This is not uncommon issue for people wanting to host the server for personal use (single user, or closed/fixed set of users). I'm happy to discuss alternative auth option.

**Comment by onyxiarivera** [comment]
> I see your point. Creating the user won't solve authentication problem alone, as you'd still need to receive email for subsequent logins.
> 
> This is not uncommon issue for people wanting to host the server for personal use (single user, or closed/fixed set of users). I'm happy to discuss alternative auth option.

2FA would be nice or auth through system users

**Comment by ku1ik** [comment]
@onyxiarivera the latest server release includes [admin panel](https://docs.asciinema.org/manual/server/self-hosting/admin/) where you can add a user and also provides a login link for any user, allowing you to authenticate without setting up email delivery.
